.babelrc

```diff
@@ -4,8 +4,8 @@
       "@babel/preset-env",
       {
         "targets": "last 1 Firefox version, last 1 Chrome version, last 1 Safari version, last 1 Edge version",
-        "modules": "amd"
+        "modules": "commonjs"
```

Comment:
`amd` was needed with `requireJS` but, since we're using jest now, using `amd` causes all tests to fail ... well, this was the most important change, that resulted in jest working smoothly  😛

Reply:
But we didn't fully remove requirejs yet and there are several tests still depending on requirejs (via `js/profile-w3c-common.js`).

Reply:
But eventually, we'd have to change it to `commonjs` right ?

Reply:
I think no, we intend to ultimately change it into `"esmodules": true`. #1640 

Currently the blocker is Firefox not supporting dynamic imports yet.

Reply:
But, now we have the support for es6 imports in firefox too, so this should work right?

Reply:
>we have the support for es6 imports in firefox too

For static imports yes, for dynamic imports no.